egs/wsj/s5/steps/nnet3/lstm/make_configs.py

```diff
@@ -76,6 +76,8 @@ def GetArgs():
                         help="clipping threshold used in ClipGradient components, if clipping-threshold=0 no clipping is done", default=30)
     parser.add_argument("--self-repair-scale", type=float,
                         help="A non-zero value activates the self-repair mechanism in the sigmoid and tanh non-linearities of the LSTM", default=None)
+    parser.add_argument("--self-repair-scale-clipgradient", type=float,
```

Comment:
It might be better to modify the defaults for both the  `--self-repair-scale` and ``--self-repair-scale-clipgradient` arguments as you find both these helpful.